app/views/training_modules/show.html.slim

```diff
@@ -7,7 +7,10 @@
   .govuk-grid-row
     .govuk-grid-column-full
       h1.govuk-heading-l= "Module #{@training_module.id}: #{@training_module.title}"
-      p.govuk-caption-m= translate_markdown(@training_module.description)
+      = translate_markdown(@training_module.description)
```

Comment:
Thinking about preparation for CMS integrity and validations, please could update the demo modules yaml and put some markdown in the fields that allow it. 

It would be good to enforce this using validations in Contentful. Could you test this when you are ready to see if this works and if so add it to the migrations.

<img width="841" alt="Screenshot 2023-02-21 at 12 00 29" src="https://user-images.githubusercontent.com/3261/220339761-eeac231b-1654-456a-82f3-cf477b9dc5b1.png">